span/daq.py

```diff
@@ -119,7 +123,7 @@ def write(self, voltages, *channels):
             sleep(samples / self.samplerate + 1/1000)
             writeTask.stop()
     
-    def readwrite(self, voltages, readChannels, writeChannels):
+    def readwrite(self, voltages: np.ndarray, readChannels, writeChannels, timeout: float = 300):

```

Comment:
`readChannel` and `writeChannel` can both be `str` or `list[str]`, so same type hints as `_addOutputChannels` and `_addInputChannels`